.github/workflows/tag_latest_image.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: Tag Latest Image
+on:
+  pull_request:
+    branches:
+    - main
+    - master
+    - develop
+    types:
+    - closed
+
+jobs:
+  docker_tag:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out GitHub Repo
+      if: github.event_name == 'pull_request' && github.event.action == 'closed' &&
+       github.event.pull_request.merged == true
+      with:
+        ref: "${{ github.event.pull_request.head.sha }}"
+      uses: actions/checkout@v2
+    -
+      name: Log in to the Container registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.GHCR_USERNAME }}
+        password: ${{ secrets.GHCR_TOKEN }}
+    -
+      name: Set up environment
+      run: |
```

Comment:
this combines the two version tagging scripts into a single streamlined GitHub action